### DISCUSSION TITLE
How to use modchart editor?

### DISCUSSION THREAD

**Discussion by kipchin1** [glowsoony/SC-SP-ENGINE#38] [General]
How to use modchart editor?

**Comment by glowsoony** [comment]
Press 6 in any song.

**Comment by glowsoony** [comment]
![image](https://github.com/EdwhakKB/SC-SP-ENGINE/assets/84847356/72916412-c920-4dc8-b737-0d8acea30d3d)
by default the in controls menu in the options menu, it contains key 3, what ever that key is (by default is 6) press during any song, and the editor will appear, if you are asking how to use @EdwhakKB could assist.
